instance/serializers/openedx_instance.py

```diff
@@ -150,7 +150,7 @@ def to_representation(self, obj):
         """
         output = super().to_representation(obj)
         output['appservers'] = [
-            AppServerBasicSerializer(appserver, context=self.context).data for appserver in obj.appserver_set.all()
+            AppServerBasicSerializer(appserver, context=self.context).data for appserver in obj.appserver_set.all()[:5]
```

Comment:
I feel like this number should be configurable.

Reply:
Ok, I made this into a variable, and we can set it to 5, 3, or any number.

Reply:
Awesome, thanks!